zebra/zebra_vty.c

```diff
@@ -333,19 +339,16 @@ DEFUN (show_ip_rpf_addr,
 }
 
 /* Static route configuration.  */
-DEFPY(ip_route, ip_route_cmd,
+DEFPY(ip_route,
+      ip_route_cmd,
       "[no] ip route\
-          <A.B.C.D/M$prefix|A.B.C.D$prefix A.B.C.D$mask>\
-          <\
-            {A.B.C.D$gate|INTERFACE$ifname}\
-            |null0$ifname\
-            |<reject|blackhole>$flag\
-          >\
-          [{\
-            tag (1-4294967295)\
-            |(1-255)$distance\
-            |vrf NAME\
-            |label WORD\
+	<A.B.C.D/M$prefix|A.B.C.D$prefix A.B.C.D$mask>                        \
+	<A.B.C.D$gate|INTERFACE$ifname|Null0$flag|reject$flag|blackhole$flag> \
```

Comment:
Now we are losing the ability to configure fully specified static routes (with both a nexthop address and nexthop interface). We need to keep `A.B.C.D$gate|INTERFACE$ifname` under curly braces. The same applies to the IPv6 DEFPY.